mos_tests/functions/os_cli.py

```diff
@@ -52,6 +53,9 @@ def __call__(self, action, flags='', params='', prefix='', fail_ok=False,
     def details(self, output):
         return {x['Field']: x['Value'] for x in json.loads(output)}
 
+    def details_table(self, output):
```

Comment:
In tempest details returns a dict, and listing - return a list of dict. Let's rename it to `listing` (and perhaps move to subclass - see comment below)